Tasks/GoTool/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "gotool",
+  "version": "0.0.1",
```

Comment:
Start with 0.1.  not patching nothing.

Reply:
Done
changes incorporated in https://github.com/Microsoft/vsts-tasks/pull/6307